### ISSUE TITLE
Please provide a doc that helps migrate 1.5.6 code to the latest version

### ISSUE THREAD

**Issue by inder123** [akshay2211/PixImagePicker#276]
**Is your feature request related to a problem? Please describe.**
We used 1.5.6 version in our code and just call Pix.start(@myActivityContext, options) and use onActivityResult() method to handle the response. I can't figure out how to convert this code to the latest 1.6.3 version since there is no activity (similar to Pix) that I can call. 

**Describe the solution you'd like**
The options could be: 
1. A clear doc explaining how to rewrite the old code to fit the new usage
2. A sample activity that is equivalent to Pix that we just call

Thanks

**Comment by duadhruv** [comment]
@inder123 did you figure out the migration?

**Comment by inder123** [comment]
No. I thought of taking the Pix activity from the previous version and rewriting it with the new one, but didn't get around to do it.

**Comment by duadhruv** [comment]
```
addPixToActivity(R.id.container, options) {
          when (it.status) {
              PixEventCallback.Status.SUCCESS -> //use results as it.data
              PixEventCallback.Status.BACK_PRESSED -> // back pressed called
          }
      }
```

Were you able to convert this to a java code?
